compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -10580,7 +10579,7 @@ private BLangRecordLiteral rewriteMappingConstructor(BLangRecordLiteral mappingC
                 BLangSimpleVarRef varRefField = (BLangSimpleVarRef) field;
                 rewrittenFields.add(ASTBuilderUtil.createBLangRecordKeyValue(
                         rewriteExpr(createStringLiteral(pos,
-                                StringEscapeUtils.unescapeJava(varRefField.variableName.value))),
+                                Utils.unescapeJava(varRefField.variableName.value))),
```

Comment:
Better to refactor as follows
```
rewriteExpr(createStringLiteral(pos, Utils.unescapeJava(varRefField.variableName.value))),
```

Reply:
Please check other places as well to see if we can refactor them in this manner.

Reply:
Got it. I'll refactor relevant places.